src/modules/analytics/AnalyticsEvent.js

```diff
@@ -1,20 +1,31 @@
 // @flow
 
+export type noop = (...args: any) => {};
```

Comment:
should return `void` not `{}` right?

Reply:
Yeah, I just got rid of it since it's a subset of the FireAnalyticsEvent type (and flow was therefore complaining about it).